homeassistant/components/keba/sensor.py

```diff
@@ -65,62 +85,39 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class KebaSensor(SensorEntity):
     """The entity class for KEBA charging stations sensors."""
 
-    def __init__(self, keba, key, name, entity_type, icon, unit, device_class=None):
+    _attr_should_poll = False
+
+    def __init__(
+        self,
+        keba,
+        entity_type,
+        description: SensorEntityDescription,
+    ):
         """Initialize the KEBA Sensor."""
         self._keba = keba
-        self._key = key
-        self._name = name
+        self._key = description.key
```

Comment:
```suggestion
```

This is not needed, us  `self.entity_description.key` instead where `self._key` is used